Symfony/CS/ConfigInterface.php

```diff
@@ -59,6 +59,8 @@ public function getFixers();
      * Sets the root directory of the project.
      *
      * @param string $dir The project root directory
+     *
+     * @return Config The same instance
```

Comment:
i checked https://github.com/phpDocumentor/phpDocumentor2/issues/514 and https://github.com/symfony/symfony/pull/13586; do you think ```@return $this``` would do it?